priorityQueue/main.c

```diff
@@ -0,0 +1,58 @@
+﻿#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "queue.h"
+
+bool test1(void)
+{
+    Queue* priorityQueue = createQueue();
+
+    ErrorCode errors[5] = { enqueue(&priorityQueue, 1, 3),
+    enqueue(&priorityQueue, 2, 1),
+    enqueue(&priorityQueue, 3, 2),
+    enqueue(&priorityQueue, 4, 5),
+    enqueue(&priorityQueue, 5, 4) };
+
+    for (size_t i = 0; i < 5; ++i)
+    {
+        if (errors[i] != ok)
+        {
+            deleteQueue(&priorityQueue);
+            return false;
+        }
+    }
+
+    int* comparativeArray[] = { 2, 3, 1, 5, 4 };
```

Comment:
Это массив указателей на int-ы, что? :) Кажется, должно быть
```suggestion
    int comparativeArray[] = { 2, 3, 1, 5, 4 };
```